src/ShellEvents.ts

```diff
@@ -1,7 +1,6 @@
 export type EventType =
   | 'V1.REQUIRE_CONTEXT'
   | 'V1.REQUIRE_AUTHENTICATION'
-  | 'V1.CLOSE'
```

Comment:
@kistll I believe we are not using this one, but I don't understand why it was added in the first place.
If you believe this change could be somehow destructive please let me know.